packages/core/src/test-utils/llm-mock.ts

```
@@ -185,31 +185,29 @@ export class MockProvider extends MastraLLMV1 {
 
-  // @ts-ignore
+  // @ts-expect-error
   stream(...args: any): PromiseLike<StreamReturn<any, any, any>> {
-    // @ts-ignore
+    // @ts-expect-error
     const result = super.stream(...args);
 
     return {
       ...result,
-      // @ts-ignore on await read the stream
       then: (onfulfilled, onrejected) => {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)